src/prr.rs

```diff
@@ -41,6 +41,11 @@ struct PrrConfig {
     ///
     /// Useful for enterprise instances with custom URLs
     url: Option<String>,
+
+    /// Activate experimental PR metadata support. Currently this option
+    /// just activates downloading the actual PR description in addition
+    /// to the diff.
+    activate_pr_metadata_experiment: Option<bool>,
```

Comment:
Nit: I think `Option<bool>` is same as `bool` with `#[serde(default)]`. Might be nicer to work with than an option